package.json

```diff
@@ -25,6 +25,26 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+          {
+            "id": "assistantChat",
+            "title": "Assistant Chat",
```

Comment:
Can we brand this with our name and just call it "CodeMaker AI".

Reply:
Will fix